lib/publiccloud/provider.pm

```diff
@@ -19,7 +19,7 @@ use Data::Dumper;
 use Mojo::JSON 'decode_json';
 
 use constant TERRAFORM_DIR     => '/root/terraform';
-use constant TERRAFORM_TIMEOUT => 17 * 60;
+use constant TERRAFORM_TIMEOUT => 17 * 60 * get_var('TIMEOUT_SCALE', 1);
```

Comment:
there is special function wrapper to cover this . I think better to use it - scale_timeout

Reply:
You're right, I will change this.